.erb-lint.yml

```diff
@@ -15,8 +15,6 @@ linters:
     rubocop_config:
       inherit_from:
         - .rubocop.yml
-        # Uncomment the following line to ignore known issues
```

Comment:
Are these linter files the unused configs you're referring to? I'd really rather have that in a separate pull since it's so completely unrelated. (I think we can skip making an issue and merge it straight away, though.)

Reply:
Yup that's it. I've split them out into https://github.com/otwcode/otwarchive/pull/4866